tests/testees/Makefile

```diff
@@ -0,0 +1,13 @@
+CC       = rustc
+CC_FLAGS = -g -C relocation-model=dynamic-no-pic
```

Comment:
This default option is likely to be reconsidered in the future, as we'll need to account for PIC anyway, but at this time we should leave this at `dynamic-no-pic` for the sake of simplicity.